apex/contrib/gpu_direct_storage/__init__.py

```diff
@@ -6,6 +6,11 @@ def GDSFile(filename, mode):
     assert type(filename) == str
     assert type(mode) == str
     try:
+        from apex import deprecated_warning
+
+        deprecated_warning(
+            "`gpu_direct_storage.GDSFile` is deprecated and will be removed in September 2025."
+        )
```

Comment:
@Aidyn-A Could you suggest a more informative message by e.g. providing pointers to the replacements?